app-backend/api/src/main/resources/application.conf

```diff
@@ -50,6 +50,9 @@ auth0 {
 }
 
 s3 {
+  region = "us-east-1"
+  region = ${?AWS_DEFAULT_REGION}
```

Comment:
We're using [`1.11.92` inherited from GeoTrellis](https://github.com/locationtech/geotrellis/blob/49fb27e36c82625a9a82243a1fcd3be0231bfc26/project/Dependencies.scala#L39). I got `AWS_DEFAULT_REGION` from the [Jenkinsfile](https://github.com/azavea/raster-foundry/blob/develop/Jenkinsfile#L10).

Will switch to `AWS_REGION` since that seems to be more standard.